### PR TITLE
Fixed the box shadow of performance tests run

### LINKED ISSUE #905: Change current box shadow.
#### Description
Change current box shadow and add the box shadow used on SS above.
Screenshot 2022-11-25 230608

#### Enviroment:
 - OS: Mac Linux Windows
 - Browser: Chrome Safari Firefox
 - Version: 
 - Device: Desktop Mobile

---
### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)

The meshery.io website uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/meshery/meshery.io/tree/master).
- 📚 See the [Contributing to Meshery.io Website](https://github.com/meshery/meshery.io#contributing-to-the-mesheryio-website) section of the readme.md and other [contributing instructions](https://docs.meshery.io/project/contributing), too.
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by Savio629**
**Description**
Fixed the box shadow of performance test run 

This PR fixes #905

**Screenshots**
Before : 

https://user-images.githubusercontent.com/91362589/233695264-1664e0ad-96a6-41c5-835d-0ede8c81c306.mp4

After :

https://user-images.githubusercontent.com/91362589/233695283-69ff75f0-54e5-4b14-a5e3-028b9ec6847e.mp4

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by Savio629** [comment]
@leecalcote @GaganpreetKaurKalsi @Chadha93

**Review by GaganpreetKaurKalsi — APPROVED** [review]
Looks good 👍

**Review by thisiskaransgit — COMMENTED** [review]
@Savio629  The change looks good, remove the white space and we are good to go

**Comment by Savio629** [comment]
@thisiskaransgit 
You mean to decrease the box width?

File: collections/_pages/features.html [hunk 1]
```diff
@@ -171,7 +171,7 @@ <h2 style="margin: 0">Use the <span class="special-text">Service Mesh Performanc
     });
   </script>
   <div class="flex"
-    style="box-shadow:  0px 0px 20px 3px rgba(71,126,150,.5);margin-top: 20px; display: flex; align-items: center; justify-content: center; clear:both; height: 40px;border-radius: 10px; margin:auto; width:90%; margin-top: 11px;">
+    style=" box-shadow: var(--box-shadow-primary);margin-top: 20px; display: flex; align-items: center; justify-content: center; clear:both; height: 40px;border-radius: 10px; margin:auto; width:90%; margin-top: 11px;">
```
Comment (thisiskaransgit) [inline collections/_pages/features.html:174]:
```suggestion
    style="box-shadow: var(--box-shadow-primary);margin-top: 20px; display: flex; align-items: center; justify-content: center; clear:both; height: 40px;border-radius: 10px; margin:auto; width:90%; margin-top: 11px;">
```
:)

**Comment by Savio629** [comment]
@thisiskaransgit done :)

**Review by thisiskaransgit — APPROVED** [review]
Looks good to merge, thank you for your contribution :tada:

**Pull request merged**